### ISSUE TITLE
Get our lighthouse scores to "good" in relevant categories

### ISSUE THREAD

**Issue by choldgraf** [executablebooks/sphinx-book-theme#468]
### Description

In #467 we've added an audit with Lighthouse that will help catch some major regressions in performance, accessibility, etc. The scores for pass/fail are lower than ideal.  We should make the necessary changes to get our lighthouse score up to some minimal threshold.

Deciding on a threshold is somewhat arbitrary, but according to [this guide](https://web.dev/performance-scoring/#color-coding), 90 is considered "good" in a category. To that extent, I think our goal should be to achieve a score of 90 in the "accessibility", "performance", and "best practices" categories.

### Implementation guide

- Some of the scores might be hit by dependencies of this theme, and we could look into making improvements there as well.
- We should consider adding one of the heavier pages (e.g. the notebooks page?) if we want to push this further

**Comment by choldgraf** [comment]
I'm going to close this in #478 - we are reliably hovering around 89-90 in each category, so I've bumped our limits to 85 in the categories we test for. I think that most of the things keeping us below 90 are not in-control by this theme (e.g., other sphinx extensions), so let's say 85 is good enough for now and spot-check improvements as we can
